ext/winevt/winevt_channel.c

```diff
@@ -254,6 +254,7 @@ rb_winevt_channel_each(VALUE self)
     }
 
     status = is_subscribable_channel_p(hChannelConfig, winevtChannel->force_enumerate);
+    EvtClose(hChannelConfig);
```

Comment:
It's not called when an error is raised before it.

Reply:
Oh! :scream:

Reply:
I've fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/23/commits/f56e348da90539bb56ba622e4a2b1b10c5e60942.